sass/_vendor.scss

```diff
@@ -1,30 +1,30 @@
-// imported from node_modules
+// imported from node_modules...
 
 @import 'normalize.css/normalize';
 
 // basscss
 
-// components (comment out if unneeded)
+// components
+//
 @import 'basscss-sass/scss/btn';
 @import 'basscss-sass/scss/btn-outline';
 @import 'basscss-sass/scss/btn-primary';
-@import 'basscss-sass/scss/forms';
-// @import 'basscss-sass/scss/highlight-dark';
-// @import 'basscss-sass/scss/highlight';
-// @import 'basscss-sass/scss/input-range';
-// @import 'basscss-sass/scss/media-object';
-// @import 'basscss-sass/scss/progress';
 
 // core utilities
-@import 'basscss-sass/scss/basscss';
+//
+@import 'basscss-sass/scss/type-scale';
```

Comment:
i did not add or remove any of the core utility modules (i just included them explicitly for visibility, instead of referring the the `basscss` file which included the same files)